ArticleTemplates/assets/js/bootstraps/liveblog.js

```diff
@@ -130,6 +130,22 @@ function onLiveMoreClick(liveMoreElem) {
     signalDevice('showmore');
 }
 
+function setupTryLive() {
+    const elems = document.getElementsByClassName('live-promo__button');
+    for (let i = 0; i < elems.length; ++i) {
+        let elem = elems[i];
+        elem.addEventListener('touchstart', () => {
+            elem.classList.add('pressed');
+        });
+        elem.addEventListener('touchend', () => {
+            elem.classList.remove('pressed');
+        });
+        elem.addEventListener('click', () => {
+            signalDevice('try-live');
```

Comment:
I think this might only be for iOS so we would need a platform check and probably a new JS interface in Android? @jordanterry

Reply:
I'll investigate this as soon as I get around to working on this

Reply:
`signalDevice` is cross-platform, Android just need to respond to the same event. This is already used for the "show more" button so you can take this as example.

Reply:
So essentially I think we can merge this safely, since the card wouldn't be shown on Android for now anyway, and when it's integrated, Jordan can make sure that's handled as expected.